records/2023-11-24.system-status.md

```diff
@@ -61,9 +61,34 @@ The website that shows system status needs to be deployed separately from the ma
 A new front-end website will be deployed to show the status information.  It will be served from a new domain, status.notification.canada.ca
 
 ### Back-end
-A new back-end API wil be deployed that will be responsible for assessing the status of each component of the system.  
+A new back-end API wil be deployed that will be responsible for assessing the status of each component of the system.
 
-#### Status checks
+### Deployment
+
+#### Api Code
+We have two options to deploy the API's associated with the system architecture page.
+1. API-lambda  
+    We can change the current heartbeat API lambda to make all the API calls required for the system status page.
+1. ECS Cluster  
+    We can set up an ECS task (similar to Blazer/Perf-tests) to allow all the API calls
+
+Given we use both versions in our current code base and neither has any significant advantage over the other, we are going to use the API lambda (Option 1)
+
+The API lambda needs to accomplish the following things:
```

Comment:
Maybe we could just add a line that this needs to be a scheduled lambda, since our front end won't interact with this API.